common/src/main/java/com/wynntils/features/user/overlays/ObjectivesOverlayFeature.java

```diff
@@ -87,7 +87,7 @@ protected GuildObjectiveOverlay() {
 
         @Override
         public void render(PoseStack poseStack, float partialTicks, Window window) {
-            WynnObjective guildObjective = ObjectiveHandler.getGuildObjective();
+            WynnObjective guildObjective = ObjectiveListener.getGuildObjective();
```

Comment:
Will this be ObjectiveModel when it is merged?

Reply:
Yes.